test/e2e/opm_test.go

```diff
@@ -251,14 +271,15 @@ var _ = Describe("opm", func() {
 			Expect(err).NotTo(HaveOccurred())
 
 			By("exporting an index to disk with containerd")
-			err = exportWith(containertools.NoneTool.String())
+			err = exportWith(containertools.NoneTool.String(), indexImage3)
 			Expect(err).NotTo(HaveOccurred())
 
 			By("loading manifests from a containerd-extracted directory")
 			err = initialize()
 			Expect(err).NotTo(HaveOccurred())
 		})
 
+		/* TODO: Reenable this test once the commutativity bug is fixed.
```

Comment:
Could you change this to https://pkg.go.dev/github.com/onsi/ginkgo?tab=doc#Skip instead?